machine_management/creating_machinesets/creating-machineset-aws.adoc

```diff
@@ -9,7 +9,7 @@ toc::[]
 You can create a different compute machine set to serve a specific purpose in your {product-title} cluster on Amazon Web Services (AWS). For example, you might create infrastructure machine sets and related machines so that you can move supporting workloads to the new machines.
```

Comment:
Maybe this part should be a snippet too? It seems to repeat a lot with only the cloud service differing.

Reply:
I am working on reorganizing this entire book (that's how I noticed the snippet was misclassified), this assembly might not really exist after I am done and rest assured I am looking to greatly reduce the amount of repeated text via snippets as I redo this :nerd_face: